modules/codelite/codelite_project.lua

```diff
@@ -379,12 +385,16 @@
 		_p(3, '</AdditionalRules>')
 	end
 
-	function m.isCpp11(cfg)
-		return (cfg.cppdialect == 'gnu++11') or (cfg.cppdialect == 'C++11') or (cfg.cppdialect == 'gnu++0x') or (cfg.cppdialect == 'C++0x')
+	function m.isCpp17(cfg)
+		return (cfg.cppdialect == 'gnu++17') or (cfg.cppdialect == 'C++17') or (cfg.cppdialect == 'gnu++1z') or (cfg.cppdialect == 'C++1z')
 	end
 
 	function m.isCpp14(cfg)
-		return (cfg.cppdialect == 'gnu++14') or (cfg.cppdialect == 'C++14') or (cfg.cppdialect == 'gnu++1y') or (cfg.cppdialect == 'C++1y')
+		return m.isCpp17(cfg) or (cfg.cppdialect == 'gnu++14') or (cfg.cppdialect == 'C++14') or (cfg.cppdialect == 'gnu++1y') or (cfg.cppdialect == 'C++1y')
```

Comment:
I find this to be a very strange change. Why does C++17 need to be viewed as C++14 and C++11?

Reply:
well c++17 has all the features of both c++14 and c++11 and c++14 has all the features of c++11 this is for code completion so before if i had c++17 the c++14 code completion would never be enabled even though it has all the features that was my logic anyway

Reply:
Just trying to understand this a bit more:
- What happens to the code completion for C++17 features? You've added `isCpp17` but that doesn't get used to tell CodeLite you're using C++17.
- Without your changes, if you were using C++14, does that mean _only_ features added in C++14 have code completion? This is kind of why I find this change so strange.